base/namedtuple.jl

```diff
@@ -44,6 +44,16 @@ julia> collect(pairs(x))
  :a => 1
  :b => 2
 ```
+
+A named tuple can be created programatically by splatting an iterator yielding
```

Comment:
programmatically? I'm not a native speaker, but I couldn't find programatically with a single M.

https://www.merriam-webster.com/dictionary/programmatically